src/Process/MemoryLimitSetter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Meteor\Process;
+
+/**
+ * Class MemoryLimitSetter
+ *
+ * @author Jadu Ltd.
+ */
```

Comment:
Unnecessary doc block

Reply:
Interesting, I ran this against `jadu/php-style` and all it wants to do is add a full stop at the end?

Reply:
Happy to remove these by the way, I have literally no preference on this cosmetic stuff, just wanted to make sure I understood the jadu/php-style properly

Reply:
I don't think it's something php-style can pick up on really. In XFP we add a rule to remove all author tags as they're superfluous. We could add this to php-style but I don't know if we should or not. As for the description, you could have a genuine description explaining the class, but in the case, it's just literally the same line as the class definition it's self. You can remove these form you're PhpStorm templates so they aren't automatically added at class generation

Reply:
Excellent, cheers for clarifying Jack, will remove